eth/eventhandler/handlers.go

```diff
@@ -233,11 +233,12 @@ func (eh *EventHandler) handleShareCreation(
 		sharePublicKeys,
 		encryptedKeys,
 	)
-	if err != nil {
-		return nil, fmt.Errorf("could not extract validator share from event: %w", err)
+
+	if err != nil && !share.Metadata.InvalidSecret {
```

Comment:
is the 2nd check needed? as far as i can see that method returns no error when the secret is invalid

Reply:
oh sry, i thought that we're not returning error in `validatorAddedEventToShare` when only the share is invalid, thats what i previously meant in my comment there

can we try to do this?

i'm interested to see how it would look, we can just log the reason for invalid secret instead of returning it as error (for example `logger.Debug("invalid secret error: could not decrypt share private")`)